### DISCUSSION TITLE
API query

### DISCUSSION THREAD

**Discussion by d-teller** [ItzCrazyKns/Vane#237] [Ideas]
Hi!
I was wondering if theres an API endpoint to perform queries or am i limited to using the UO for this?

I was thinking of deploying this solution and have a custom plugin in Librechat that would forward web search inputs to Preplexica. 

Thanks

**Comment by rrubio** [comment]
API access would be awesome!

**Comment by ItzCrazyKns** [comment]
Hi, there is no API endpoint as of no. There is though a websocket implementation (the client code is inside ChatWindow.tsx file). We're working on adding an API endpoint to allow users to use Perplexica externally.
